rust/agama-dbus-server/src/network/nm/dbus.rs

```diff
@@ -412,6 +443,15 @@ fn wireless_config_from_dbus(conn: &OwnedNestedHash) -> Option<WirelessConfig> {
     Some(wireless_config)
 }
 
+fn bond_config_from_dbus(conn: &OwnedNestedHash) -> Option<BondConfig> {
+    let Some(_bond) = conn.get(BOND_KEY) else {
+        return None;
+    };
+
+    // TODO
```

Comment:
I wasn't able to convert the given value to a HashMap<String, String>. Maybe someone can help?

Reply:
Something like this?
```
    let options: zvariant::Dict = bond
        .get("options")?
        .downcast_ref::<Value>()?
        .try_into()
        .unwrap();
    let miimon: &str = options.get("miimon").unwrap()?;
```

Reply:
Almost, I don't need (and don't even know what could be there) an specific value but all.
More like this...
```
let h: HashMap<String,String> = HashMap::new();
options.iter().map(|e| h.insert(e.0 , e.1) );
```
But with correct syntax :)

Reply:
I guess something like this would be a way to generate a HashMap:
```
    let options_dict: zvariant::Dict = bond
        .get("options")?
        .downcast_ref::<Value>()?
        .try_into()
        .unwrap();
    let options: HashMap<String, String> = options_dict.try_into().unwrap();
```
although I'm not sure if there is maybe a simpler way.